README.md

```diff
@@ -213,149 +187,47 @@ For Example:
             /openldap-modify-users.yaml <- openldap overlay
  ```
 
-### Running
+## Creating and Managing deployment
 
-All configs needed by ansible are also needed to be mounted into the docker container. In general any file/folder path set via an ansible flag are equivalent to the file/folder being mounted to the docker container at /config/<lower_case_variable_name>.
+Create and manage deployments by either: 
 
-Examples:
+- running [Ansible](docs/user/AnsibleUsage.md) directly on your workstation, or
+- running [Docker container](docs/user/DockerUsage.md). 
+  
+### DNS
 
-- The ansible flag `-e KUBECONFIG` is equivalent to `--volume <path_to_file>:/config/kubeconfig` when running the docker container
-- The ansible flag `-e JUMP_SVR_PRIVATE_KEY` is equivalent to `--volume <path_to_file>:/config/jump_svr_private_key` when running the docker container
-- The ansible flag `-e V4_CFG_SITEDEFAULT` is equivalent to `--volume <path_to_file>:/config/v4_cfg_sitedefault` when running the docker container
+During the installation, an ingress loadbalancer can be installed for viya and the monitoring and logging stack. The hostname for these services must be registered with your dns provider to resolve to the loadbalancer's endpoint. This can be done by creating a record per unique ingress host. However, when managing multiple viya deployments, this could get cumbersome. In which case, we recommend creating a dns record that points to the ingress controller's endpoint. The endpoint may be an ip address or fqdn depending on the cloud provider. 
 
-Below are the only exceptions:
+- Create an A record or CNAME (depending on cloud provider) that resolves the desired hostname to the loadbalancers' endpoint. 
+- Create a wildcard CNAME record that resolves to the record created in the previous step.
+  
 
-| Ansible Flag | Docker Mount Path | Description | Required |
-| :--- | :--- | :--- | ---: |
-| -e BASE_DIR | /data | local folder in which all the generated files can be stored. If you do not wish to save the files, this can be omitted | false |
-| --vault-password-file | /config/vault_password_file | Full path to file containing the vault password | false |
+For example:
 
-#### Ansible
-
-In the example command line below, replace each of the <> values, such as "<path_to_kubeconfig_file>", with the appropriate value.
+First we lookup the ingress controller's loadbalancer endpoint. Here, we will use kubectl but we could have also checked in the cloud providers portal.
```

Comment:
"First lookup the ingress controller's loadbalancer endpoint. The example below uses kubectl. This information can also be looked up in the cloud providers portal."

Reply:
updated